refinery/ui/source/js/tool-launch/partials/tool-params.html

```diff
@@ -0,0 +1,129 @@
+<div
+  class="panel-group"
+  id="tool-info-display"
+  role="tablist"
+  aria-multiselectable="true">
+
+  <div class="panel panel-default">
+    <div class="panel-heading" role="tab">
+      <h4 class="panel-title">
+        <div
+          role="button"
+          data-toggle="collapse"
+          data-parent="#tool-info-display"
+          ng-click="$ctrl.isToolParamsCollapsed = !$ctrl.isToolParamsCollapsed"
+          aria-expanded="true">
+            <i
+              ng-class="$ctrl.isToolParamsCollapsed ? 'fa fa-caret-right' : 'fa fa-caret-down'"
+              class="fa-pull-left"
+              aria-hidden="true"></i>
+            Parameters
+        </div>
+      </h4>
+    </div>
+    <div
+      id="tool-launch-body"
+      class="panel-collapse"
+      uib-collapse="$ctrl.isToolParamsCollapsed"
+      role="tabpanel"
+      aria-labelledby="tool-body">
+      <div class="panel-body">
+       <ul class="plain">
+         <form name="toolParamForm">
+           <li ng-repeat="param in $ctrl.params">
+             <div class="refinery-subheader">
+               <div>
+                 <h3>
+                   {{ param.name}}
+                 </h3>
+               </div>
+             <div ng-if="param.is_user_adjustable">
+               <div ng-switch="param.value_type">
+                 <div ng-switch-when="INTEGER">
+                   <h4>
+                     Value:
+                   </h4>
+                   <input
+                     autofocus
+                     type="text"
+                     name="{{param.uuid}}"
+                     ng-class="{'form-validation-error': toolParamForm[param.uuid].$error.pattern}"
+                     placeholder="{{ param.default_value }}"
+                     ng-model="$ctrl.paramsForm[param.uuid]"
+                     ng-maxlength="32"
+                     ng-pattern="/^\d+$/">
```

Comment:
Yes, there are default values for params. No idea, but this pattern can be changed once we have actual parameter specs.